taskcluster/scripts/schedule-tasks.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import os
+from pprint import pprint
+
+import slugid
+import taskcluster
+from taskgraph.util.parameterization import resolve_timestamps
+from taskgraph.util.time import current_json_time
+
+INDEX_PATHS = ["gecko.v2.mozilla-central.latest.taskgraph.decision-os-integration"]
+
+if "TASKCLUSTER_PROXY_URL" in os.environ:
+    options = {"rootUrl": os.environ["TASKCLUSTER_PROXY_URL"]}
+else:
+    options = taskcluster.optionsFromEnvironment()
+
+index = taskcluster.Index(options)
+queue = taskcluster.Queue(options)
+secrets = taskcluster.Secrets(options)
+
+stage_auth = secrets.get("project/releng/fxci-config/taskcluster-stage-client")
+print(f"clientId: {stage_auth['clientId']}")
+queue_stage = taskcluster.Queue(
+    {
+        "rootUrl": "https://stage.taskcluster.nonprod.cloudops.mozgcp.net/",
+        "credentials": secrets.get(
+            "project/releng/fxci-config/taskcluster-stage-client"
+        ),
+    }
+)
+
+for path in INDEX_PATHS:
+    data = index.findTask(path)
+    assert data
+    task_id = data["taskId"]
+
+    task_graph = queue.getLatestArtifact(task_id, "public/task-graph.json")
+    assert task_graph
+
+    for label, task in task_graph.items():
+        assert isinstance(task, dict)
+        if task.get("attributes", {}).get("unittest_variant") != "os-integration":
+            continue
+
+        task_def = task["task"]
+        task_def["priority"] = "low"
+        del task_def["routes"]
```

Comment:
Presumably you need to get rid of dependencies here too?
But then need a way to handle things like docker images and fetches, that reference other task ids that would not exist on staging?

Reply:
Heh, yeah good call.

My rough idea was to re-write the `MOZ_FETCHES` environment variable to use `static-url` instead. But haven't looked too closely at that yet.